framework/sandstone_p.h

```diff
@@ -357,7 +357,6 @@ struct SandstoneApplication : public InterruptMonitor, public test_the_test_data
         std::array<Slices, 2> plans;
     };
 
-    using PerThreadFailures = std::vector<__uint128_t>;
```

Comment:
If possible, keep an alias here to wherever this ends up, to facilitate updating the internal code that currently uses this type.

Reply:
It can, but it'll require an update to the internal code to do so. I was hoping to leave it unchanged. You'll see the issue if you build the internal tool with this version of OpenDCDiag.